webapp-django/crashstats/topcrashers/views.py

```diff
@@ -300,6 +295,6 @@ def topcrashers(request, days=None, possible_days=None, default_context=None):
             label = option.capitalize()
         context['process_type_values'].append((value, label))
 
-    context['platform_values'] = settings.DISPLAY_OS_NAMES
+    context['platform_values'] = [item['name'] for item in platforms]
```

Comment:
It's almost tempting to turn this entire table into a constant defined in the code. I'll think about that and maybe redo this that way.

Reply:
I'll think about that another day. For now, we'll just turn it into a table.